src/actions/PrescriptionActions.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-unused-expressions */
+
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+import { batch } from 'react-redux';
+
+import { UIDatabase, createRecord } from '../database';
+
+import { PrescriberActions } from './PrescriberActions';
+import { WizardActions } from './WizardActions';
+
+export const PRESCRIPTION_ACTIONS = {
+  REFRESH: 'Prescription/refresh',
+};
+
+export const updateDirection = (id, newValue) => (_, getState) => {
+  const { prescription } = getState();
+  const { transaction } = prescription;
+  const { items } = transaction;
+
+  const item = items.filtered('id == $0', id)[0];
+  item?.setItemDirection(UIDatabase, newValue);
+};
+
+export const removeItem = id => (dispatch, getState) => {
```

Comment:
I can't see this being imported anywhere - so it doesn't need to be exported here